src/App.jsx

```diff
@@ -4,47 +4,24 @@ import ProductCard from "./components/ProductCard/ProductCard";
 import PersonProfile from "./components/PersonProfile/PersonProfile";
```

Comment:
There is an import  "Person profile" which doesnt exist anymore in main

Reply:
and the calendar toggle import is missing

Reply:
Yes. You are correct. I also got the same errors. Now the main is fixed. I updated the branch to the recent main. So now it should work.